Frontend/CarRentalAdmin/src/app/login/login.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="row">
+  <div class="col-md-6 mx-auto">
+    <div class="card rounded-0">
+      <div class="card-header">
+        <h3>Login</h3>
+      </div>
+      <div class="card-body">
+        <div *ngIf="authFailed"
+             class="alert alert-danger">
+          Login, Password or both are incorrect
+        </div>
+        <form (ngSubmit)="onSubmit()" #loginForm="ngForm">
+          <div class="form-group">
+
+            <label class="mb-0" for="login">Admin</label>
+            <input #login="ngModel" type="text" class="form-control" id="login"
+                   required minlength="4" appForbiddenName="qwerty"
+                   [(ngModel)]="admin.login" name="login" (ngModelChange)="resetAuthStatus()">
```

Comment:
```suggestion
            <input 
              #login="ngModel"
              required
              id="login"
              name="login"
              class="form-control"
              type="text"
              minlength="4"
              appForbiddenName="qwerty"
              (ngModelChange)="resetAuthStatus()"
              [(ngModel)]="admin.login" 
            >
```

when there are a lot of attributes please move each of them to separate line for better readability
it is also a good practice to use single code style for attributes order:
1. Structural directives (e.g. *ngIf)
2. Directives (e.g. header)
3. Css classes (e.g. class="class-name")
4. Property binding (e.g. [name]="name")
5. Event binding (e.g. (change)="updateSomething()")
6. Two-way binding (e.g. [(age)]="age")